__tests__/reactTags.test.js

```diff
@@ -228,6 +228,29 @@ describe('Test ReactTags', () => {
 
       expect(tags).to.deep.have.same.members(expected);
     });
+
+    test('should allow pasting text only up to maxLength characters', () => {
+      const tags = [];
+      const $el = mount(mockItem({ handleAddition(tag) {
+            tags.push(tag);
+          }, maxLength: 5 }));
+
+      const $input = $el.find('.ReactTags__tagInputField');
+
+      $input.simulate('paste', {
+        clipboardData: {
+          getData: () => 'Thimbleberry',
+        },
+      });
+
+      // Note that 'Thimbleberry' should be sub String to 'Thimb'
```

Comment:
Since the title of spec specifies that it should only allow up to `maxlength` chars so this comment can be removed.